src/utils.jl

```diff
@@ -1,6 +1,6 @@
-const _cache = Dict{String,Array}()
+const _cache = Dict{AbstractString,Array}()
```

Comment:
Fixed by https://github.com/johnmyleswhite/Languages.jl/pull/7 . Thanks for watching, @tkelman